### PR TITLE
Run `go generate` to generate the latest codecs

### PR DESCRIPTION
**Pull request by masih**
Re-generate code by running `go generate` to pick up the latest codecs
added to the codec table.

Most notable codec generated here is the `CarIndexSorted` codec needed in
go-car v2.

See:
- https://github.com/ipld/go-car
- https://github.com/ipld/go-car/milestone/1
- https://github.com/ipld/go-car/blob/wip-v2/v2/index/index.go

Relates to:
- https://github.com/multiformats/multicodec/pull/220

### CODE REVIEW

**Review by marten-seemann — APPROVED** [review]
We should probably add a GitHub Actions workflow that runs `go generate` and checks that the definitions are up to date (not required for this PR).

**Comment by masih** [comment]
> We should probably add a GitHub Actions workflow that runs `go generate` and checks that the definitions are up to date (not required for this PR).

Thank you for the speedy review. That'd be great 👍 I have captured #47 and #48 already

File: code.go [hunk 1]
```diff
@@ -7,1387 +7,1405 @@ package multicodec
...
-	// Rlp is tagged "serialization" and described by: recursive length prefix.
+	// Rlp is tagged "serialization" and described by: draft.
 	Rlp Code = 0x60 // rlp
 
-	// Bencode is tagged "serialization" and described by: bencode.
+	// Bencode is tagged "serialization" and described by: draft.
 	Bencode Code = 0x63 // bencode
 
-	// DagPb is tagged "ipld" and described by: MerkleDAG protobuf.
+	// DagPb is tagged "ipld" and described by: permanent.
```
Comment (mvdan) [inline code.go:130]: surely these are all regressions? the old description was better

Comment (masih) [reply code.go:130]: I am with you. Maybe something to improve in `go generate` or the CSV structure?

**Pull request merged**